AuxTel/Standard-Operations/Nighttime-Operations/Monitoring-LATISS-Disperser-Wheel-Position.rst

```diff
@@ -0,0 +1,144 @@
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+- The disperser wheel is (re)aligned to its nominal position of 67mm.
+- The CCD disperser coverage has increased, resulting in a higher resolution of the AuxTel spectral data.
+
+.. _Monitoring-LATISS-Disperser-Wheel-Position-Procedure-Steps:
+
+Procedure Steps
+===============
+
+1. Access RubinTV and navigate to the AuxTel image display page.
+
+2. At the beginning of night observations, select the spectrum from a sequence number with the following characteristics:
+  
+   a. **Image Type:** science
+
+   b. **Filter:** empty
+  
+   c. **Grating/Disperser:** holo4_003
+
+3. Locate the large atmospheric absorption line due to oxygen (:math:`\lambda = 762 \, \text{nm}`, see :ref:`Fig 2 <figure-Linear-Stage-Positions>`).
+
+.. note::
+    The oxygen absorption line is always visible, but may be difficult to locate for certain spectra. 
+    Make sure to verify the absorption profile is present in both the 1D and 2D spectra. 
+    If it seems too difficult to identify in the spectra then try another sequence.
+
+4. Estimate the distance in pixels from the beginning of the spectrum to the peak of the absorption line. 
+   Label this value :math:`D(O_2)`.
+
+5. Compare the estimated value with the expected value :math:`\boxed{D(O_2) \sim 1200 \pm 100 \, \text{pixels.}}`
+
+.. _figure-Linear-Stage-Positions:
+
+.. list-table:: Fig 2: Spectra of incorrect (left) and correct (right) alignment of the disperser wheel.
+   :width: 100%
+   :class: borderless
+
+   * - 
+      .. image:: _static/LATISS_BadLinearStagePos.png
+          :width: 100%
+         
+     - 
+      .. image:: _static/LATISS_GoodLinearStagePos.png
+          :width: 100%
+
+
+.. _Monitoring-LATISS-Disperser-Wheel-Position-Troubleshooting:
+
+Troubleshooting
+===============
+
+.. This section should include troubleshooting information. Information in this section should be strictly related to this procedure.
+
+.. If there is no content for this section, remove the indentation on the following line instead of deleting this sub-section.
+
+     No troubleshooting information is applicable to this procedure.
+
+- If the distance is smaller than expected (:ref:`Fig 2 <figure-Linear-Stage-Positions>`), follow instructions on :ref:`ATSpectrograph Recovery <LATISS-Troubleshooting-ATspectrograph-Recovery>`.
```

Comment:
I don't think we should recommend jumping straight into the ATSpectrograph Recovery unless there is a fault with the spectrograph. We should instead recommend to first try running the latiss_checkout.py script which will attempt to reposition the grating stage. If that fails, then we should move on to the recovery procedure.